src/ray/raylet/object_manager_integration_test.cc

```diff
@@ -206,7 +206,7 @@ class TestObjectManagerIntegration : public TestObjectManagerBase {
                   << "\n";
     ClientTableDataT data;
     gcs_client_2->client_table().GetClient(client_id_1, data);
-    RAY_LOG(INFO) << (ClientID::from_binary(data.client_id) == ClientID::nil());
+    RAY_LOG(INFO) << (ClientID::from_binary(data.client_id) == ClientID());
```

Comment:
above we used .is_nil() for this, we might want to unify for consistency

Reply:
Thanks!